com.regnosys.rosetta.lib/src/main/java/model/basictypes.rosetta

```diff
@@ -6,6 +6,7 @@ basicType int
 basicType number
 basicType boolean
 basicType time
+basicType any
```

Comment:
I think we removed the need for the `any` type, by keeping one-of and choice rules as language features. So we should either remove the basic type here or have validation to stop usage outside of the core library.

Reply:
You are right! Just forgot to remove. I will replace it with `empty` later